pkg/core/function.go

```diff
@@ -327,6 +327,11 @@ func (c *client) waitForSuccessOrFailure(namespace string, name string, gen int6
 						someStateIsUnknown = true
 						break
 					}
+					// ignore any "scaled to zero" status messages - wait for the revision to start
```

Comment:
I assume we can remove this after https://github.com/knative/serving/issues/2346 is resolved

Reply:
Added a TODO